sktime/classification/shapelet_based/_rocket_classifier.py

```diff
@@ -70,11 +76,13 @@ def __init__(
         ensemble=False,
         ensemble_size=25,
         random_state=None,
+        n_jobs=None,
     ):
         self.num_kernels = num_kernels
         self.ensemble = ensemble
```

Comment:
Convention is to use `n_estimators=None` rather than `ensemble_size`, which could default to 1 if not given, making the boolean ensemble option redundant

Reply:
Renaming of the parameter will introduce potential incompatibility with the existing code. Should we have both parameters as of now and a `DeprecationWarning` that the old parameter will be removed in future releases?

Reply:
Yes deprecating it would be nice (we don't have a policy on deprecation cycles at the moment, too much is still in flux at this stage)